terraform/modules/ecs/task-definitions/sync.json.tftpl

```diff
@@ -85,7 +85,11 @@
       "command": [
         "CMD-SHELL", 
         "wget --no-verbose --tries=1 --spider http://localhost:8000/services/collector/health > /dev/null || exit 1"
-        ]
+        ],
+        "interval":30,
```

Comment:
If we don't explicitly set these AWS will consider our task definitions as having drifted which cause noisy "update in place" resources